org.osate.ui/src/org/osate/ui/wizards/AadlProjectWizard.java

```diff
@@ -320,17 +308,23 @@ public boolean performFinish() {
 		selectAndReveal(newProject);
 		final IProject p = getNewProject();
 
-		String filepath = p.getFile(WorkspacePlugin.AADLPATH_FILENAME).getRawLocation().toString();
```

Comment:
Please remove this and the following constants from WorkspacePlugin. All of them shouldn't be used anywhere else. If they are, that's wrong.